internal/pkg/config/default.go

```diff
@@ -350,6 +350,9 @@ func setDefaultConfigValuesWithViper(v *viper.Viper, b *BgpConfigSet) error {
 	}
 
 	for idx, server := range b.BmpServers {
+		if server.Config.SysName == "" || server.Config.SysDescr == "" {
+			return fmt.Errorf("BMP Initiation Message must contain sysDescr and sysName Information TLVs")
+		}
```

Comment:
What are appropriate default strings for them? I prefer to set something appropriate if not specified.

Reply:
RFC1213
`sysDescr OBJECT-TYPE
              SYNTAX  DisplayString (SIZE (0..255))
              ACCESS  read-only
              STATUS  mandatory
  DESCRIPTION
                      "A textual description of the entity.  This value
                      should include the full name and version
                      identification of the system's hardware type,
                      software operating-system, and networking
                      software.  It is mandatory that this only contain
                      printable ASCII characters."`
`sysName OBJECT-TYPE
              SYNTAX  DisplayString (SIZE (0..255))
 ACCESS  read-write
              STATUS  mandatory
              DESCRIPTION
                      "An administratively-assigned name for this
                      managed node.  By convention, this is the node's
                      fully-qualified domain name."`

Reply:
I think SysName could have the default value, but I'm not sure SysDescr .